### PR TITLE
Added transfers and paypal accounts endpoints

### PR DESCRIPTION
**Pull request by aseveryn-epam**

### CODE REVIEW

File: hyperwallet/api.py [hunk 1]
```diff
@@ -1450,6 +1452,212 @@ def deactivatePaperCheck(self,
 
     '''
 
+    Transfers
+    https://portal.hyperwallet.com/docs/api/v3/resources/transfers
+
+    '''
+
+    def createTransfer(self,
+                         data=None):
```
Comment (gmeyer-hw) [inline hyperwallet/api.py]: I cloned this project and after I execute this command **Make test** then I faced some code style warning like this: _hyperwallet/api.py:1461:26: E127 continuation line over-indented for visual indent ... _

File: hyperwallet/tests/test_api.py [hunk 2]
```diff
@@ -1163,6 +1163,105 @@ def test_deactivate_paper_check_success_with_notes(self, mock_post):
 
     '''
 
+    Transfers
```
Comment (gmeyer-hw) [inline hyperwallet/tests/test_api.py]: if you run **make coverage** then you can face some errors like this: 
 '''======================================================================
ERROR: test_failed_connection (hyperwallet.tests.test_client.ApiClientTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/gmeyer/IntellijWorkspace/python-sdk-aseveryn-epam/hyperwallet/tests/test_client.py", line 19, in setUp
    SERVER
TypeError: __init__() takes exactly 5 arguments (4 given)

======================================================================
ERROR: test_receive_non_json_response (hyperwallet.tests.test_client.ApiClientTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/gmeyer/IntellijWorkspace/python-sdk-aseveryn-epam/hyperwallet/tests/test_client.py", line 19, in setUp
    SERVER
TypeError: __init__() takes exactly 5 arguments (4 given)'''

File: hyperwallet/api.py (hunk 1 above)
Comment (aseveryn-epam) [reply hyperwallet/api.py]: Fixed

File: hyperwallet/tests/test_api.py (hunk 2 above)
Comment (aseveryn-epam) [reply hyperwallet/tests/test_api.py]: In scope of this pull request there were no changes in apiclient constructor, but in pull request "Layer 7 encryption" number of arguments in apiclient constructor had been changed, so it seems you have an intersection of those two pull requests. Please check it again

**Pull request merged**